static-assets/components/cstudio-common/common-api.js

```diff
@@ -5079,8 +5079,8 @@ var nodeOpen = false,
              */
             lookupContentItem: function(site, path, callback, isDraft, populateDependencies) {
 
-                path = unescape(path);
-                var serviceUri = this.lookupContentItemServiceUri + "?site=" + site + "&path=" + escape(path);
+                path = decodeURI(path);
```

Comment:
Remove line. And use without encoding on next line.

Reply:
explanation comment added